admin/partials/settings/auto-update.php

```diff
@@ -420,11 +415,34 @@ function get_themes_update_markup( $auto_update_settings, $translations ) {
 
 $auto_update_markup = ' ' . get_heading_markup( $boldgrid_backup_settings, $auto_update_settings );
 
+$auto_update_markup .= '
+	<div class="bg-box">
+		<div class="bg-box-top">' .
+			esc_html__( 'Configure When Auto Updates Occur', 'boldgrid-library' ) . '
+		</div>
+	<div class="bg-box-bottom">';
+
 if ( $this->core->config->is_premium_done ) {
```

Comment:
We can decrease complexity of the code by not requiring the `$this->core->config->is_premium_done` check and just running the filter. The premium plugin isn't ran unless "is_premium_done". https://github.com/BoldGrid/boldgrid-backup/blob/master/admin/class-boldgrid-backup-admin-core.php#L832-L838

Reply:
I have refactored that section to be somewhat less complex, however the is_premium_done, because if the premium plugin IS active, and the filter doesn't return the markup, then the message to upgrade Total Upkeep is displayed, with a link to the updates page. If premium is not active, then we display the upsell markup with a link to buy Premium. However, if we do not test for Premium being installed, then people who do not have it installed will be prompted to upgrade instead of install.